autogpt/llm_utils.py

```diff
@@ -13,10 +14,54 @@
 from autogpt.types.openai import Message
 
 CFG = Config()
-
 openai.api_key = CFG.openai_api_key
 
 
+def retry_openai_api(
```

Comment:
Would love this to be less openai specific as we work towards including other LLMs

Reply:
I’d also keep this goal in mind as you refactor in general

Reply:
This commit is a mistake, it's supposed to be in a different PR.  reverting